styles/HowWeWork.module.css

```diff
@@ -0,0 +1,31 @@
+.how_we_work {
+  display: grid;
+  grid-template-columns: 69% 31%;
```

Comment:
Please refer to the CSS of Contact section to keep consistency
![image](https://user-images.githubusercontent.com/73293255/220802511-4460ff60-cba0-4523-aa34-db637a9aab55.png)